package.json

```diff
@@ -37,6 +37,7 @@
   "dependencies": {
     "@azure/openai": "^1.0.0-beta.2",
     "@codemirror/lang-markdown": "^6.1.1",
+    "@mui/icons-material": "^5.11.16",
```

Comment:
Use needle icons instead, so that we can cut this dependency and we are closer to our ultimate goal of being Needle-only :)